eraser-map.yaml

```diff
@@ -794,6 +794,16 @@ styles:
         base: points
         texture: pois
         interactive: true
+        
+    icons-overlay:
+        base: points
+        texture: pois
+        interactive: true
+        blend: overlay
```

Comment:
At present, I don't think there's any way to guarantee this in tangram-es. 

`priority` only affects the result of label collisions, and collision is turned off for routing and search results. 

`order` would conceptually fit this need, but currently it has no effect on `overlay` styles (features in these styles are all drawn at the same depth).